packages/tools/replay-tool/src/replayMessages.ts

```diff
@@ -125,11 +125,10 @@ class Logger implements ITelemetryBaseLogger {
             // Stack is not output properly (with newlines), if done as part of event
             const stack: string | undefined = event.stack as string | undefined;
             delete event.stack;
-            console.error(`An error has been logged from ${this.containerDescription}!`);
-            console.error(event);
-            if (stack) {
-                console.error(stack);
-            }
+            const err = `An error has been logged from ${this.containerDescription}!\n
+                        ${JSON.stringify(event)}\nStack:\n${stack}`;
+            // throw instead of printing an error to fail tests
+            throw new Error(err);
```

Comment:
I think you can do something like that:
```
error = new Erorr("...");
error.stack = stack;
throw error;
```
maybe casting to any is required, not sure

Reply:
I thought I had seen code like that somewhere, but I can't find it now.